### PR TITLE
Add osu!catch Project Loved Discord link

### PR DESCRIPTION
**Pull request by wesley-221**
Added osu!catch Project Loved Discord link to the articles, used the osu! > osu!taiko > osu!catch > osu!mania order that is used everywhere. Left alone the articles that were out of date.

### CODE REVIEW

File: wiki/Community/Project_Loved/en.md [hunk 1]
```diff
@@ -51,4 +51,5 @@ All of the organisation and planning for Project Loved is done in the `#osu-love
 In addition to the discussion channels mentioned above, osu!taiko and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
 
 - [osu!taiko Project Loved Discord server](https://discord.com/invite/GhfjtZ6)
+- [osu!catch Project Loved Discord Server](https://discord.gg/phgtyS4UCh)
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:54]:
```suggestion
- [osu!catch Project Loved Discord server](https://discord.gg/phgtyS4UCh)
```

File: wiki/Community/Project_Loved/en.md [hunk 2]
```diff
@@ -51,4 +51,5 @@ All of the organisation and planning for Project Loved is done in the `#osu-love
 In addition to the discussion channels mentioned above, osu!taiko and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
```
Comment (Walavouchey) [inline wiki/Community/Project_Loved/en.md:51]:
```suggestion
In addition to the discussion channels mentioned above, the osu!taiko, osu!catch, and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
```

File: wiki/Community/Project_Loved/en.md [hunk 3]
```diff
@@ -48,7 +48,8 @@ There are no strict rules governing which maps a captain must nominate for Loved
 
 All of the organisation and planning for Project Loved is done in the `#osu-loved` channel of the [osu!dev Discord server](https://discord.gg/ppy). There is also a `#project-loved` channel in the [osu! Community Discord server](https://discord.gg/0Vxo9AsejDkGlk3H). Any questions, concerns, or suggestions may be directed there.
 
-In addition to the discussion channels mentioned above, osu!taiko and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
+In addition to the discussion channels mentioned above, the osu!taiko, osu!catch, and osu!mania communities have their own dedicated Discord hubs, which you may join via the following links:
```
Comment (The-Last-Cookie) [inline wiki/Community/Project_Loved/en.md:51]: If this description is changed, the translations should either be updated accordingly, or get the outdated tag.

**Comment by ZeroPyrozen** [comment]
Indonesian translation is good  👍

**Review by Walavouchey — APPROVED** [review]
server icon is a little scuffed but ok

**Comment by Walavouchey** [comment]
in the future, try to not pr from the master branch. make a new branch instead

**Pull request merged**